src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.State.cs

```diff
@@ -59,23 +60,26 @@ public State(SyntaxNode interfaceNode, SyntaxNode classOrStructDecl, INamedTypeS
 
                 if (service.CanImplementImplicitly)
                 {
+                    state.UnimplementedMembersNotRequiringExplicitImplementation = state.ClassOrStructType.GetAllUnimplementedMembersNotRequiringExplicitImplementation(
+                        interfaceTypes, cancellationToken);
+
                     state.UnimplementedMembers = state.ClassOrStructType.GetAllUnimplementedMembers(
                         interfaceTypes, cancellationToken);
 
                     state.UnimplementedExplicitMembers = state.ClassOrStructType.GetAllUnimplementedExplicitMembers(
                         interfaceTypes, cancellationToken);
 
-                    var allMembersImplemented = state.UnimplementedMembers.Length == 0;
+                    var allMembersImplemented = state.UnimplementedMembersNotRequiringExplicitImplementation.Length == 0;
                     var allMembersImplementedExplicitly = state.UnimplementedExplicitMembers.Length == 0;
 
                     return !allMembersImplementedExplicitly || !allMembersImplemented ? state : null;
                 }
                 else
                 {
-                    state.UnimplementedMembers = state.ClassOrStructType.GetAllUnimplementedExplicitMembers(
+                    state.UnimplementedMembersNotRequiringExplicitImplementation = state.ClassOrStructType.GetAllUnimplementedExplicitMembers(
```

Comment:
I really don't like this name in this context. We put all unimplemented members in this bucket so that the title of the code fix will be: "Implement Interface" not "Implement Interface Explicitly", but it means we're lying about what this property contains.